quodlibet/data/operon.bash

```diff
@@ -21,6 +41,9 @@ _operon() {
     local cur prev words cword
     _init_completion -n = || return
 
+    local audio_extensions='@(a?(l)ac|ac3|ape|flac|m[4k]a|mid|mp3|og[ag]|opus|w?(a)v)'
```

Comment:
From the top of your head, any other extension we should add here?
(for the background, this list is used so that only files with these extensions are shown by the completion)

Reply:
```
>>> from quodlibet.formats import init, loaders
>>> init()
>>> loaders.keys()
dict_keys(['.dsf', '.aac', '.adif', '.adts', '.wav', '.vgm', '.wma', '.asf', '.wmv', '.wv', '.mp4', '.m4a', '.m4v', '.3gp', '.3g2', '.3gp2', '.mid', '.ape', '.mp3', '.mp2', '.mp1', '.mpg', '.mpeg', '.669', '.amf', '.ams', '.dsm', '.far', '.it', '.med', '.mod', '.mt2', '.mtm', '.okt', '.s3m', '.stm', '.ult', '.gdm', '.xm', '.aif', '.aiff', '.aifc', '.mpc', '.mp+', '.ogg', '.oga', '.flac', '.oggflac', '.spx', '.ogv', '.opus', '.spc', '.tta'])
```

Reply:
Ack, thanks! I will rework the completion script to generate this list dynamically, instead of hard-coding it.